app/src/main/java/org/mozilla/webmaker/activity/Play.java

```diff
@@ -15,10 +19,23 @@ public Play() {
         super("project", R.id.page_layout, R.layout.page_layout, R.menu.menu_play);
     }
 
+    protected String login_username;
```

Comment:
You are storing the user `id` not the `username`. Something like `sessionId` might make more sense here.